shims/spark311cdh/src/main/scala/com/nvidia/spark/rapids/shims/spark311cdh/SparkShimServiceProvider.scala

```diff
@@ -20,7 +20,7 @@ import com.nvidia.spark.rapids.{ClouderaShimVersion, SparkShims}
 
 object SparkShimServiceProvider {
   val VERSION = ClouderaShimVersion(3, 1, 1, "3.1.7270")
-  val VERSIONNAMES = Seq(s"3.1.1.3.1.7270")
+  val VERSIONNAMES = Seq(s"3.1.1.3.1.7270.0-253")
```

Comment:
why this change?

Reply:
Without this change, the Shim for CDH is never loaded as the version in the pom.xml is `3.1.1..3.1.7270.0-253` which doesn't match `3.1.1.3.1.7270` so `SparkShimServiceProvider.VERSIONNAMES.contains(version)` returns false. 

I don't know how it was working previously